engines/python/setup/djl_python/utils.py

```diff
@@ -51,29 +51,25 @@ def parse_input_with_formatter(
         try:
             content_type = item.get_property("Content-Type")
             input_map = decode(item, content_type)
+            _inputs, _param, is_client_side_batch[i] = _parse_inputs_params(
+                input_map, item, input_format_configs)
+            if input_format_configs.is_adapters_supported:
+                adapters_per_item, found_adapter_per_item = _parse_adapters(
+                    _inputs, input_map, item, adapter_registry)
+                adapters.extend(adapters_per_item)
+                found_adapters = found_adapter_per_item or found_adapters
         except Exception as e:  # pylint: disable=broad-except
```

Comment:
Should this really `except Exception`? Maybe it should be more targeted than that. (Note: I recognize it was like that before this PR).

Reply:
May be the idea was to suppress any kind of exception? Otherwise, we should target specific exceptions which is not in the scope for this PR.

Reply:
Yeah but, suppressing any exception and trying to just continue the loop is bad.

If it's "not in the scope of this PR", then I guess I should create an Issue for stuff like this?